src/src/settings.py

```diff
@@ -145,11 +145,9 @@
 
 AUTHENTICATION_BACKENDS = [
     'social_core.backends.open_id.OpenIdAuth',
-    'social_core.backends.google.GoogleOpenId',
     'social_core.backends.google.GoogleOAuth2',
     'social_core.backends.google.GoogleOAuth',
     'social_core.backends.twitter.TwitterOAuth',
-    'social_core.backends.yahoo.YahooOpenId',
     'social_core.backends.github.GithubOAuth2',
     'django.contrib.auth.backends.ModelBackend',
     'rest_framework_social_oauth2.backends.DjangoOAuth2',
```

Comment:
Thanks for the PR. Do we need to use other auth backends? We only use Github right now. Login with other medium is blocked I think. So is it better to remove all these dependencies and clear out code in python3?

Reply:
Yes, I agree. We could remove the additional Twitter and Google backends, in case we have no plans to provide these logins in the future.

Reply:
Yeah lets remove unnecessary auth backends

Reply:
All done, working as expected.